Microsoft.Toolkit.Uwp.UI.Controls/MarkdownTextBlock/Parse/Inlines/ImageInline.cs

```diff
@@ -116,14 +122,30 @@ internal static Common.InlineParseResult Parse(string markdown, int start, int e
                 pos++;
             }
 
-            string url = TextRunInline.ResolveEscapeSequences(markdown, urlStart, pos);
+            var imageWidthPos = markdown.IndexOf(" =", urlStart, pos - urlStart, StringComparison.Ordinal);
```

Comment:
If we're going to add this we should support both Width and Height for the full `=WxH` pattern that's being suggested in the CommonMark standard.  I believe patterns could be `=W`, `=WxH`, or `=xH`.

Reply:
@michael-hawker what ImageStretch mode should we use when both height and width are specified?

Reply:
How about uniform?